dev/source/docs/sitl-with-jsbsim.rst

```diff
@@ -0,0 +1,41 @@
+.. _sitl-with-jsbsim:
+
+======================
+Using SITL with JSBSim
+======================
+
+JSBSim is a sophisticated flight
+simulator that is used as the core flight dynamics system for several
+well known flight simulation systems.
+
+In the past ArduPilot required a special version of JSBSim. As of
+December 2018 that is no longer the case, and we can use the
+standard JSBSim releases.
+
+In order to build JSBSIM, you need Cmake, install it with :
+
+::
+
+    sudo apt-get install cmake
+
+In the same directory (your home directory) run these commands:
+
+::
```

Comment:
We still have `Tools/scripts/build-jsbsim.sh` - why aren't we pointing to it here, I wonder...

Reply:
Added ref